receiver/awscontainerinsightreceiver/internal/k8swindows/extractors/extractorhelpers.go

```diff
@@ -14,12 +52,35 @@ func ConvertPodToRaw(podStat *stats.PodStats) *RawMetric {
 
 	if podStat.CPU != nil {
 		rawMetic.Time = podStat.CPU.Time.Time
-		rawMetic.CPUStats = podStat.CPU
+		rawMetic.CPUStats = convertCPUStats(podStat.CPU)
 	}
 
 	if podStat.Memory != nil {
-		rawMetic.MemoryStats = podStat.Memory
+		rawMetic.MemoryStats = convertMemoryStats(podStat.Memory)
+	}
+	return rawMetic
+}
+
+// ConvertContainerToRaw Converts Kubelet Container stats per Pod to RawMetric.
+func ConvertContainerToRaw(containerStat *stats.ContainerStats, podStat *stats.PodStats) *RawMetric {
+	var rawMetic *RawMetric
```

Comment:
same feedback.  I think the pointer return type just adds complexity to the code.  And the pointer input types compel us to null check before dereferencing

Reply:
Addressed, I have removed pointer references.